Paragraph/Primary/index.jsx

```diff
@@ -54,11 +54,16 @@ Primary.propTypes = {
 }
 
 export default compose(
-  themeable((customizations, props) => ({
-    style: {
-      ...props.style,
-      color: customizations.color_text
+  themeable((customizations, props) => {
+    if (['error', 'warning'].includes(props.color)) {
```

Comment:
Yeah, you're right that the distinction needs to be done in the adapter. My bad. But we could still have a function that takes an adapter and returns a new adapter where the distinction can be done. Something like:

```
themeable(ifThemeableState(() =>({
  style: {
    ...props.style,
    color: customizations.color_header
  }
})))
```

Where `ifThemeableState` is:

```
export default (adapter) => (props) => ['error', 'warning'].includes(props.color) ? {} : adapter(props)
```